File: Leetcode/1. Two Sum/java/Main.java
public class Main {
    public static void main(String[] args) {
        // 初始化数组
        int[] nums = {2, 7, 11, 15};

        // 求解
        Solution solution = new Solution();
        int[] result = solution.twoSum(nums, 9);

        // 打印
        for (int x: result) {
            System.out.printf("%d ", x);
        }
    }
}

File: Leetcode/1. Two Sum/cpp/main.cpp
#include <iostream>
#include <vector>

#include "twoSum.hpp"

using namespace std;


int main() {
    // 初始化数组
    vector<int> nums = {2, 7, 11, 15};

    // 求解
    vector<int> result = Solution().twoSum(nums, 9);

    // 打印
    for (int x: result) {
        cout << x << " ";
    }

    return 0;
}